tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -188,6 +197,23 @@ Verify Favorite Resource Cards
     Favorite Items Should Be Listed First When Sorted By    ${favorite_ids}    duration
     [Teardown]    Remove Items From favorites    @{favorite_ids}
 
+Verify Notifications Are Shown When Notebook Builds Have Not Started
+    [Documentation]     Verifies that Notifications are shown in RHODS Dashboard when Notebook builds haven't started
+    [Tags]    Sanity
+    ...       ODS-1347
+    ...       Tier1
```

Comment:
As the test took 48 minutes, could you add this tag? `Execution-Time-Over-30m`

Reply:
Updated